src/renderers/WebGLRenderer.js

```diff
@@ -2080,47 +2080,90 @@ function WebGLRenderer( parameters = {} ) {
 
 	};
 
-	this.copyTextureToTexture = function ( position, srcTexture, dstTexture, level = 0 ) {
+	this.copyTextureToTexture = function ( srcTexture, dstTexture, dstPosition = undefined, srcBox = undefined, dstLevel = 0, srcLevel = 0 ) {
+
+		if ( ! srcTexture.isTexture ) {
+
+			console.warn( 'THREE.WebGLRenderer: .copyTextureToTexture() now expects (srcTexture, dstTexture, dstPosition, srcBox, dstLevel, srcLevel).' );
+			return this.copyTextureToTexture( arguments[ 1 ], arguments[ 2 ], arguments[ 0 ], undefined, arguments[ 3 ] );
+
+		}
+
+		const image = srcTexture.mipmaps[ srcLevel ] || ( srcLevel == 0 ? srcTexture.image : null );
```

Comment:
Hum, how should we differentiate base texture from mipmap 0? @mrdoob
Here what's happening is a bit weird to me

Reply:
That makes sense actually. If you have mipmaps, level 0 should be the normal textures... But that's confusing